.github/workflows/OCV-Android-SDK.yaml

```diff
@@ -0,0 +1,82 @@
+name: OCV PR:4.x AndroidSDK
```

Comment:
> .github/workflows/OCV-Android-SDK.yaml

"4.x" should be part of the file name if workflow is named as 4.x.

> github.base_ref == '3.4'

these checks are not needed then.

Reply:
I propose to use universal pipeline for all branches, if it's possible. Let's just remove branch name here.

Reply:
We have different NDK for different branches.
3.4 didn't use ndk18 (probably something like 8-10).
5.x should use something modern than ndk18.
So even build images are different.